packages/dashboard/src/constants/wallet-labels.ts

```diff
@@ -1,6 +1,6 @@
 import { localize } from '@deriv/translations';
 
-export const getWalletLabels = () => ({
+export const getWalletLabels = (): any => ({
```

Comment:
It returns an object with a specific structure, please type it rather than using `any`.